.github/workflows/ci.yml

```diff
@@ -62,7 +62,7 @@ jobs:
 
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
```

Comment:
This should probably go to a separate PR.

Reply:
I don't think this is a big thing. I just wanted get rid of the annoying warnings under the Github actions. See the blog post: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/

**Changelog** of actions/checkout v3: https://github.com/actions/checkout/blob/main/CHANGELOG.md

> ## v3.0.0
>    
> Update to node 16 https://github.com/actions/checkout/pull/689

**Warnings** in Github Actions: https://github.com/simplepie/simplepie/actions/runs/4025417866

![grafik](https://user-images.githubusercontent.com/2162994/215799475-c5f64de8-d624-4678-8f8f-c25dd68bad4a.png)

Reply:
I reverted the version change and opend #783